src/io/flutter/view/FlutterView.java

```diff
@@ -91,6 +91,8 @@ public void dispose() {
   protected static final String INSTALLATION_IN_PROGRESS_LABEL = "Installing JxBrowser and DevTools...";
   protected static final String INSTALLATION_TIMED_OUT_LABEL =
     "Waiting for JxBrowser installation timed out. Restart your IDE to try again.";
+  protected static final String INSTALLATION_STOPPED_LABEL =
+    "The JxBrowser installation was stopped for unknown reasons. Restart your IDE to try again.";
```

Comment:
We should word smith this error message. 
How about
"The JxBrowser installation failed unexpectedly.  Restart your IDE to try again."
also change the label to 
JX_BROWSER_INSTALLATION_FAILED
to clarify this is a failure.

Reply:
This is slightly different because it's `installation.get()` that threw an exception, but the message makes sense.